Directory.Build.props

```diff
@@ -72,7 +72,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'netcoreapp3.1'">
-    <LangVersion>11</LangVersion>
+    <LangVersion>12</LangVersion>
```

Comment:
Do we want to continue to update this manually moving forward or use 'preview' instead to always be up to date?

Reply:
With preview don't we risk that a preview compiler produces binaries which are not right? Would they be meant for prod?

Reply:
There's that risk but didn't we used "latest" before which technically had the same risk? also, it does allow us to catch any issues from a new C# version earlier and not later.

Reply:
"latest" did not infer the same risk and maps to pre-11